RecoMET/METProducers/src/HcalNoiseInfoProducer.cc

```diff
@@ -125,6 +124,19 @@ HcalNoiseInfoProducer::HcalNoiseInfoProducer(const edm::ParameterSet& iConfig) :
   for(int i = 0; i < 128; i++)
      adc2fC[i] = adc2fCTemp[i];
 
+  // adc -> fC for qie8 with PMT input, for laser monitor
+  const float adc2fCTempHF[128]={-3,-0.4,2.2,4.8,7.4,10,12.6,15.2,17.8,20.4,23,25.6,28.2,30.8,33.4,
```

Comment:
hi @jkunkle - i don't think that this intermediate variable is needed. you should be able to just do

adc2fCHF = {..}

otherwise, please add some documentation of the source of these numbers (that should be in a conditions db..)